examples/14_react-tweet/vite.config.ts

```diff
@@ -0,0 +1,10 @@
+/** @type {import('vite').UserConfig} */
+export default {
+  optimizeDeps: {
+    include: ['react-tweet'],
+  },
+  ssr: {
+    external: ['use-sync-external-store'],
+    noExternal: ['react-tweet'],
```

Comment:
`use-sync-external-store` is cjs. 

`react-tweet` is externalized and that means vite will not resolve it using our plugins and will directly fallback to nodejs module system. What this means is that we'll `.module.css` error since it's not natively supported by nodejs.